### ISSUE TITLE
java.lang.NullPointerException

### ISSUE THREAD

**Issue by Zubayer011** [Drumber/RemoteLight#10]
This issue was automatically created by the RemoteLight ExceptionHandler.
RemoteLightCore version: v0.2.4-rc1
Java version: 1.8.0_261
OS: Windows 10
StackTrace:
```
java.lang.NullPointerException
	at de.lars.remotelightcore.RemoteLightCore.showNotification(RemoteLightCore.java:269)
	at de.lars.remotelightcore.RemoteLightCore.showErrorNotification(RemoteLightCore.java:287)
	at de.lars.remotelightcore.RemoteLightCore.<init>(RemoteLightCore.java:125)
	at de.lars.remotelightclient.Main.<init>(Main.java:90)
	at de.lars.remotelightclient.Main.main(Main.java:78)

```
*Aug 31, 2020 10:54 PM*

**Comment by Drumber** [comment]
Hey, thanks for your bug report! Since this bug has been occupying me for quite some time and I cannot reproduce the cause, I would have 2 more questions:
- Does the error occur every time RemoteLight is started, or only sometimes?
- Are all settings and outputs deleted after this error occurs?

I would appreciate it if you could send me the log file that corresponds to the error.
You can find it in `C:\Users\%YourUserName%\.RemoteLight\logs`

Thanks in advance.

**Comment by snowsign** [comment]
It happens for me every time I start the app. It shows the splash screen, then crashes.

**Comment by Drumber** [comment]
Hi, thanks for your reply. Could you check if the folder `.RemoteLight` exists in the `C:\Users\%yourUsername%\` directory.
Should look something like this:
![.RemoteLight folder](https://user-images.githubusercontent.com/29163322/91868152-b8dbde80-ec74-11ea-8c29-ea866bd2ddac.png)
  
If the folder exists, is there a data.json file in it? If so, could you check if the file is read or write protected.

![data file properties](https://user-images.githubusercontent.com/29163322/91868042-977af280-ec74-11ea-85c9-cfd7a132ed9b.png)

I assume that RemoteLight cannot access the directory. Do you have an AntiVirus or anything else that could block the file access?

**Comment by snowsign** [comment]
My permissions look identical to yours. I don't have any sort of antivirus, and it feel like it would be a much bigger issue if it was being picked up by windows defender. It seems like there's some malformed JSON at the end of 
[the file](https://github.com/Drumber/RemoteLight/files/5157410/data.json.txt) (I had to change the extension so that GitHub would accept it.)

**Comment by Drumber** [comment]
Oh, I see what happened. The file was not completely written to disk. Deleting data.json should at least make RemoteLight usable for you.
I will try to find and fix the bug before the next release.
Thanks for sending, it helped me a lot. 👍

**Comment by Drumber** [comment]
Just released the new version that hopefully fixes the bug.
\> https://github.com/Drumber/RemoteLight/releases/tag/v0.2.4-rc2

**Comment by Drumber** [comment]
Hey, I would like to know if the bug was fixed with v0.2.4-rc2 so I can close this issue.
